### PR TITLE
Update s8bbs.com.xml

### PR DESCRIPTION
**Pull request by stlwvikx**

### CODE REVIEW

**Comment by gloomy-ghost** [comment]
Thanks. Please:

- [ ] Move s8bbs.com into a [mixed-content](https://github.com/EFForg/https-everywhere/blob/master/docs/rulesets.md#mixed-content-blocking-mcb) ruleset.

Note as `Refused` in a top comment:
- [ ] attach.s8bbs.com
- [ ] content.s8bbs.com

File: src/chrome/content/rules/s8bbs.com.xml [hunk 1]
```diff
@@ -1,20 +1,24 @@
...
 	<target host="adp.s8bbs.com" />
 		<test url="http://adp.s8bbs.com/g.php" />
 	<target host="adv.s8bbs.com" />
 		<test url="http://adv.s8bbs.com/adv_top.htm" />
-	<target host="attach.s8bbs.com" />
-		<test url="http://attach.s8bbs.com/attachments/Mon_1408/84_13572009_d9a94b0e1d11f6f.jpg" />
-	<target host="content.s8bbs.com" />
-		<test url="http://content.s8bbs.com/ad/newbbs/20160805_769_989.gif" />
 	<target host="hot.s8bbs.com" />
 		<test url="http://hot.s8bbs.com/g.php" />
```
Comment (gloomy-ghost) [inline src/chrome/content/rules/s8bbs.com.xml]: Also please edit the test url to http://adp.s8bbs.com/g.php?id=3&t=g

**Comment by stlwvikx** [comment]
https://github.com/stlwvikx/https-everywhere/blob/23ae54df72f7171a08f4c08920b8bda4578b0ced/src/chrome/content/rules/s8bbs.com.xml#L10

Delete this and create new ruleset?

**Comment by gloomy-ghost** [comment]
Yes, you can move [L8-L16](https://github.com/stlwvikx/https-everywhere/blob/23ae54df72f7171a08f4c08920b8bda4578b0ced/src/chrome/content/rules/s8bbs.com.xml#L8-L16) into a separate mixed content ruleset without any exclusions. Also please link the rulesets in their top comments after create a new one.

**Comment by stlwvikx** [comment]
Misunderstand.
Tor is important than Chrome/Firefox/Opera?

**Comment by gloomy-ghost** [comment]
For Chrome/Firefox/Opera we have to leave the page request in http otherwise all http resources in the page will get blocked before https-everywhere has a chance to redirect them. IMO only redirecting external resources to https isn't a big improvement on security, and Tor users are quite sensitive about the security. Keep the redirecting rule for Chrome/Firefox/Opera users if you want, but can you also create a mixedcontent ruleset so that Tor can open the page in full https?

**Comment by stlwvikx** [comment]
https://github.com/EFForg/https-everywhere/blob/master/docs/rulesets.md#mixed-content-blocking-mcb

I have read this style guide again and again.
I could not find what you said.
Add the 'Tor Firstly' to it, then I will creat mixcontent ruleset.

**Comment by ivysrono** [comment]
@stlwvikx Maintain the exclusion and create s8bbs.com-mixcontent.xml
```
<!--
	Releated:
		s8bbs.com.xml
-->
<ruleset name="s8bbs.com-mixcontent" platform="mixedcontent">
	<target host="s8bbs.com" />

	<rule from="^http:" to="https:" />
</ruleset>
```
We should do for users the more the better, both Chrome/Firefox/Opera/Tor, and so on.

**Pull request merged**